plugins/TodayAtMun/plugin.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "TodayAtMun",
+  "version": "0.2.0",
```

```suggestion
  "version": "0.1.0",
```
I assume you mean version `0.1.0`, since this is a fresh plugin :)